guides/v2.0/cloud/configure/cloud-vcl-custom-snippets.md

```diff
@@ -59,7 +59,7 @@ You can use the following key/value pairs in JSON snippets in VCL files and in `
     <td>priority</td>
     <td>
       <p>Determines the order VCL snippets call. Lower values run first, from `1` to `100`. All uploaded snippets from a Magento module have a value of `50`. If you want an action to occur last or to override Magento default VCL snippets, use a higher number, such as `100`. To have code occur immediately, use a lower value, such as `5`.</p>
-      <p>Any VCL snippet with a priority value of `5` runs immediately, which is best for blacklists, whitelists, and redirects. Priority `100` is best for overriding default VCL snippet code and for extending timeouts. If you do not set a priority with your cURL command, the default value set is `100`.</p>
+      <p>Any VCL snippet with a priority value of `5` runs immediately, which is best for blacklists, whitelists, and redirects. Priority `100` is best for overriding default VCL snippet code. If you do not set a priority with your cURL command, the default value set is `100`.</p>
```

Comment:
You can still use a VCL to extend timeouts for other pages.  Not sure if that's worth metioning since we are no longer providing the specific example but it's definitely something that can still be done.

Reply:
I think I'll leave it out since we don't provide the snippet anymore.